src/EditorFeatures/Test2/Utilities/TestNotificationService.vb

```diff
@@ -0,0 +1,33 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Notification
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
+    Friend Class TestNotificationService
+        Implements INotificationService
```

Comment:
Amusingly, we already had 3 test implementations of this! Will switch to the INotificationServiceCallback approach. Thanks!